[CI] ClusterInfoServiceIT testMaxQueueLatenciesInClusterInfo failing

**Build Scans:**
- [elasticsearch-periodic #10466 / openjdk23_checkpart1_java-matrix](https://gradle-enterprise.elastic.co/s/j4xssbdj3cetg)
- [elasticsearch-periodic #10436 / encryption-at-rest](https://gradle-enterprise.elastic.co/s/4ykucyykslozy)
- [elasticsearch-periodic-platform-support #10421 / rhel-9_platform-support-unix](https://gradle-enterprise.elastic.co/s/6xnylecrhk7l6)
- [elasticsearch-periodic-platform-support #10421 / almalinux-9_platform-support-unix](https://gradle-enterprise.elastic.co/s/eythkfekqiqvm)
- [elasticsearch-pull-request #89327 / part-1](https://gradle-enterprise.elastic.co/s/yrdau4y3quioe)
- [elasticsearch-intake #27818 / part1](https://gradle-enterprise.elastic.co/s/cooqdjhnd4alc)

**Reproduction Line:**
```
./gradlew ":server:internalClusterTest" --tests "org.elasticsearch.cluster.ClusterInfoServiceIT.testMaxQueueLatenciesInClusterInfo" -Dtests.seed=14C0ED9EF970F45D -Dtests.locale=kkj-Latn-CM -Dtests.timezone=GMT0 -Druntime.java=23
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.cluster.ClusterInfoServiceIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('testMaxQueueLatenciesInClusterInfo'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Expected: <0.0F>
     but: was <0.014456825F>
```

**Issue Reasons:**
- [main] 6 failures in test testMaxQueueLatenciesInClusterInfo (0.9% fail rate in 661 executions)
- [main] 2 failures in pipeline elasticsearch-periodic (18.2% fail rate in 11 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/muted-tests.yml
+++ b/muted-tests.yml
@@ -516,9 +516,6 @@ tests:
 - class: org.elasticsearch.health.HealthPeriodicLoggerTests
   method: testOutputModeNoLogging
   issue: https://github.com/elastic/elasticsearch/issues/134200
-- class: org.elasticsearch.cluster.ClusterInfoServiceIT
-  method: testMaxQueueLatenciesInClusterInfo
-  issue: https://github.com/elastic/elasticsearch/issues/134088
 - class: org.elasticsearch.xpack.esql.qa.single_node.GenerativeForkIT
   method: test {csv-spec:spatial_shapes.ConvertFromStringParseError}
   issue: https://github.com/elastic/elasticsearch/issues/134254

